Fix:

error: call to undeclared function 'select'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
--- test/test.h.orig	2023-07-15 01:06:00.000000000 -0500
+++ test/test.h	2023-12-28 01:14:35.000000000 -0600
@@ -378,6 +378,7 @@
   #define THREAD_SLEEP( ms ) Sleep( ms )
   typedef unsigned ( __stdcall *THREAD_FUNC )( void *arg );
 #elif defined( UNIX_THREADS )
+  #include <sys/time.h>
   #define THREAD_HANDLE		pthread_t
   #define THREAD_EXIT()		pthread_exit( ( void * ) 0 )
   #define THREAD_SELF()		pthread_self()
